### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by Rozygeldi94**

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[[redirects]]
+from = "/*"
+to = "index.html"
+status = 200
+
+
```
Comment (mskmee) [inline netlify.toml:6]: we dont need 2 redirects setting options. It's confusing

File: src/pages/SingleProduct/index.tsx [hunk 2]
```diff
@@ -38,6 +41,18 @@ export const SingleProductPage = () => {
   const [isAllReviewsBtnVisible, setIisAllReviewsBtnVisible] = useState(
     currentProduct?.[0]?.reviews.length > 2 ? true : false,
   );
+  const maxLength = 500;
```
Comment (mskmee) [inline src/pages/SingleProduct/index.tsx:44]: declare this constant outside of the component function

File: src/routes.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    children: [
+      { path: AppRoute?.ROOT, element: <Main /> },
+      { path: AppRoute?.SMARTPHONES, element: <Smartphones /> },
+      { path: AppRoute?.LAPTOPS, element: <Laptops /> },
+      { path: AppRoute?.VIEWED, element: <Viewed /> },
+      {
+        path: AppRoute?.SINGLE_PRODUCT,
+        element: <SingleProductPage />,
+      },
+    ],
```
Comment (mskmee) [inline src/routes.tsx:23]: you don't need optional chaning if your are use enums

File: vite.config.ts [hunk 4]
```diff
@@ -4,7 +4,7 @@ import react from '@vitejs/plugin-react';
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
-  base: './',
+  base: '/',
```
Comment (mskmee) [inline vite.config.ts:7]: this one should brake `npm build` why do you change it?

**Review by mskmee — CHANGES_REQUESTED** [review]

File: vite.config.ts (hunk 4 above)
Comment (Rozygeldi94) [reply vite.config.ts:7]: there is a problem on the product page. when i go to the product page and reload the page, there is a problem, only a white screen appears. therefor i changed vite.config.ts , routes.tsx  and i created netlify.toml .

**Review by mskmee — APPROVED** [review]

**Pull request merged**